include/internal/statem.h

```diff
@@ -6,6 +6,8 @@
  * in the file LICENSE in the source distribution or at
  * https://www.openssl.org/source/license.html
  */
+#ifndef STATEM_H
```

Comment:
Do we need this guard on an internal header?

It's not the correct name for the guard: `OSSL_INTERNAL_STATEM_H`

Reply:
It was added because it is needed.